public/static/js/admin.js

```diff
@@ -147,6 +147,20 @@ mciModule.controller('AdminSettingsController', ['$scope', '$window', '$http', '
       });
   }
 
+  $scope.clearCommitQueues = function(){
+    if(!confirm("This will clear the contents of all commit queues. Continue?")){
+      return
+    }
+
+    $http.delete('/admin/commit_queues').then(
```

Comment:
If you move the route to restV2, could you add this route to rest.js and call it from there?

Reply:
done.